rust/lance-index/src/lib.rs

```diff
@@ -67,7 +67,14 @@ pub trait Index: Send + Sync + DeepSizeOf {
 #[derive(Debug, PartialEq, Eq, Copy, Hash, Clone, DeepSizeOf)]
 pub enum IndexType {
     // Preserve 0-100 for simple indices.
-    Scalar = 0,
+    Scalar = 0, // BTree
```

Comment:
Can we add BTree and let Scalar still indicate BTree?
That could be more clear

Reply:
Ok, add one